genie-client/src/integTest/java/com/netflix/genie/client/JobClientIntegrationTest.java

```diff
@@ -143,6 +145,34 @@ void canSubmitJob() throws Exception {
         final String dateJobId = this.jobClient.submitJob(dateJob);
         final String echoJobId = this.jobClient.submitJob(echoJob);
 
+        final Map<String, JobStatus> expectedStatuses = ImmutableMap.<String, JobStatus>builder()
+            .put(sleepJobId, JobStatus.SUCCEEDED)
+            .put(killJobId, JobStatus.KILLED)
+            .put(timeoutJobId, JobStatus.KILLED)
+            .put(dateJobId, JobStatus.SUCCEEDED)
+            .put(echoJobId, JobStatus.SUCCEEDED)
+            .build();
+
+        final long waitStart = System.currentTimeMillis();
+        final long maxTotalWait = 120000;
+
+        for (final Map.Entry<String, JobStatus> entry : expectedStatuses.entrySet()) {
+            final String jobId = entry.getKey();
+            final JobStatus status = entry.getValue();
+
+            System.out.println("Waiting for job: " + jobId + " (expected final status: " + status.name() + ")");
```

Comment:
`log.warn` or even `info` seems better than leaving `System.out` in.

Reply:
The reason for stdout here is that it shows up in the test reports. So when smoke tests fail we can at least learn the job ID and look at server logs. Whereas lines printed with slf4j logger are forever lost.